bin/cli.js

```diff
@@ -31,6 +33,15 @@ Options
       auth: {
         type: "string",
       },
+      httpHeaders: {
+        type: "string",
```

Comment:
What do you think about passing header values as an array like curl does that? I guess it will be easier to write headers one by one in cli instead of making one json.

How curl does that: `curl -header "X-First-Name: John" --header "X-Last-Name: Doe" http://example.com/`

Also probably with this way we won't need to parse headers because `meow` could handle multiple values for a flag: https://github.com/sindresorhus/meow#flags

Reply:
Is this a block or to getting merged? Or can we add this at a later point?￼￼

Reply:
I definitely feel that this is a very useful feature and it has a way cleaner API, but can this be an additional option? I think both ways would be valid ways of inputting HTTP headers￼￼

Reply:
@mbelsky I just realized a unique edge case on why this might not be the best API for the user. Currently, the values are the only thing stored in the array. Therefore, we would have to enforce a strict string schema of ‘-x "headerKey:headerValue‘ and then split on the string. However, compared to a JSON object (yes, definitely more verbose), they will potentially run into issues with spacing and casing. We would need much more strict validation

Reply:
@drwpow What do you think about this implementation? I personally think that we should be okay to merge and then think about this additional feature and implementation next.

Reply:
We have [discussed](https://github.com/drwpow/openapi-typescript/issues/645#issuecomment-868938899) with @drwpow about this feature. Just wanna to link the issue to this pr.

Reply:
I can add this as an additional alternative￼

Reply:
@mbelsky and @drwpow We now support both methods. Please check the updated PR.